### ISSUE TITLE
End to End Test with NFC automatically created

### ISSUE THREAD

**Issue by Surerocks** [richardperryman/NFC-Engineering-Project#15]
So the NFC is being saved on the server correctly after a login (and I have tested that it checks access correctly as well). So it is ready to be tested with the end to end process of logging in on the phone and tapping the lock to open.

To test, user 'Craig' with password 'password' is good to login, and will give you access to lock with lock_id=1

@richardperryman Is the app ready to be tested by @jessicamorris ?

**Comment by Surerocks** [comment]
In case it wasn't clear, the version on AWS (v0.4.5.2) has all the changes to work now.

**Comment by richardperryman** [comment]
Yes the app is ready! I committed it a while ago, but since I couldn't test the service no promises that works.

**Comment by jessicamorris** [comment]
Done adulting for the day, trying to log in with the app looks really weird on my end... This is what the login screen looks like to me, when I tried a tap the app crashed:
![screenshot_2017-02-23-15-29-17](https://cloud.githubusercontent.com/assets/12285757/23277789/37f72fc8-f9dd-11e6-83d0-8b6365352b5c.png)

**Comment by richardperryman** [comment]
Hmm that's odd, let me try to figure that out. I've got a meeting with a professor soon, but I'll let you know when I figured it out.

**Comment by richardperryman** [comment]
Alright, somehow I swapped the wrong less than and greater than signs without noticing when I checked in a while ago... they should be fixed now.

**Comment by richardperryman** [comment]
Or they would be if my laptop was able to resolve github.com? If I can't work this out making the change yourself should be easy, it's in `MainActivity.java` and in the `login` method. The `if` statement should have a `auth.login > 0` instead of `auth.login < 1`.

**Comment by richardperryman** [comment]
Ok, the changes should be up, sorry for the spam, weird internet troubles :(

**Comment by jessicamorris** [comment]
Hahaha sounds good, I will take another shot with it!

**Comment by jessicamorris** [comment]
App success has been achieved :tada:

**Comment by jessicamorris** [comment]
HTTPS appears working on my end, do we have a user I could use to test for "access denied"?

**Comment by Surerocks** [comment]
Any other user should get access denied on lock_id 1 I think
